README.md

```diff
@@ -46,15 +46,23 @@ Quick Start
 You need Python 2.6 or 2.7. Python 3+ is not currently supported.
 
 To install, we recommend using [pip](https://pip.pypa.io/) and a
-[virtualenv](https://virtualenv.pypa.io/en/stable/). In Debian-based systems:
+[virtualenv](https://virtualenv.pypa.io/en/stable/).
+
+In Debian-based systems:
 
 	sudo apt install python-setuptools
 	sudo -H easy_install pip
 	sudo -H pip install virtualenv
+	virtualenv env --no-site-packages
```

Comment:
the `--no-site-packages` flag is deprecated (it's now the default behavior), so I don't think it should be on here. Did you have a specific issue with this on your Arch system?